gdpr/impl_test.go

```diff
@@ -178,12 +178,13 @@ func TestMalformedConsent(t *testing.T) {
 
 func TestAllowPersonalInfo(t *testing.T) {
 	tests := []struct {
-		description string
-		bidderName  openrtb_ext.BidderName
-		publisherID string
-		gdpr        Signal
-		consent     string
-		allowPI     bool
+		description         string
+		bidderName          openrtb_ext.BidderName
+		publisherID         string
+		userSyncIfAmbiguous bool
```

Comment:
I like this change. Could you please add `userSyncIfAmbiguous: false` to the tests which have it disabled? I know this is the default value and not strictly necessary, but it makes the test cases more self documenting and easier to read.